### PR TITLE
[EPO-2752] GalaxiesSelector supports selections

### PR DESCRIPTION
**Pull request by blnkt**
Includes minor updates to GalaxySElector & GalacticProperties styles;
Includes update to QAAccordion padding to match other card-style QAs;
Includes updated hsc data;
Includes qa content update to count selected galaxies;
Includes some duplicated changes from previous commit (accidental);

### CODE REVIEW

File: src/components/charts/galacticProperties/Points.jsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ class Points extends React.PureComponent {
           });
           const blueColorPercent = Math.floor((color / 2) * 100);
           const redColorPercent = 100 - blueColorPercent;
-          const colorShift = `rgb(${redColorPercent}%, 0%, ${blueColorPercent}%)`;
+          const colorShift = `rgba(${redColorPercent}%, 0%, ${blueColorPercent}%, 0.85)`;
```
Comment (georgegomez144) [inline src/components/charts/galacticProperties/Points.jsx]: Love the use of the transparency here!!!!

File: src/containers/GalaxiesSelectorContainer.jsx [hunk 2]
```diff
@@ -43,19 +47,47 @@ class GalaxiesSelectorContainer extends React.PureComponent {
...
+    );
+
     return (
       <>
         <h2 className="space-bottom">Galaxies Selector</h2>
         <div className="galaxies-selector-images--container">
           <GalaxySelector
             className="galaxies-selector"
+            {...{ selectedData }}
             data={data}
```
Comment (georgegomez144) [inline src/containers/GalaxiesSelectorContainer.jsx]: Why not include `data={data}` in the spead above??? something like `{...{ selectedData, data }}`??

File: src/containers/WithQAing.jsx [hunk 3]
```diff
@@ -105,6 +106,13 @@ export const WithQAing = ComposedComponent => {
       return `${numOfGalaxies} galaxies & ${numOfSupernovae} supernovae`;
     }
 
+    getGalaxiesContent(data) {
+      const numGalaxies = Object.keys(data).length;
+      return `${numGalaxies} ${
+        numGalaxies > 1 ? 'galaxies' : 'galaxy'
```
Comment (georgegomez144) [inline src/containers/WithQAing.jsx]: Maybe also do `numGalaxies > 1 || numGalaxies === 0 ? 'galaxies' : 'galaxy'`
Not sure if we'll ever have 0 galaxies, but at least if we did the text would show correctly?

Comment (blnkt) [reply src/containers/WithQAing.jsx]: Good catch, but in this case content is not rendered unless there is an answer present (a selection), and you are not able to "select" 0 things so we can confidently assume numGalaxies >= 1

File: src/containers/GalaxiesSelectorContainer.jsx (hunk 2 above)
Comment (blnkt) [reply src/containers/GalaxiesSelectorContainer.jsx]: true enough: will make this change and amend it to [EPO-2790](https://github.com/lsst-epo/investigation-static/pull/85)

File: src/containers/WithQAing.jsx (hunk 3 above)
Comment (georgegomez144) [reply src/containers/WithQAing.jsx]: Awesome! That's what I figured, but just wanted to be certain.

**Pull request closed without merging**